bindings/terra/legion_terra.h

```diff
@@ -16,7 +16,7 @@
 #ifndef __LEGION_TERRA_H__
 #define __LEGION_TERRA_H__
 
-#include "legion_c.h"
+#include "legion.h"
```

Comment:
Again, same as before. This is externally facing pure-C functionality. Do not include C++ headers from this file.